applications/SwimmingDEMApplication/python_scripts/variables_management.py

```diff
@@ -14,7 +14,7 @@ def GetGlobalVariableByName(variable_name):
     modules = [Kratos, DEM, SDEM]
     for mod in modules:
         try:
-            return getattr(mod, variable_name)
+            return getattr(mod, variable_name) # this could use KratosMultiphysics.KratosGlobals.GetVariable(var_name)
```

Comment:
@GuillermoCasas Do you agree with the change?

Reply:
I mean, do we change it according to the comment added by @philbucher ?

Reply:
example:
https://github.com/KratosMultiphysics/Kratos/blob/d78596fcfcf82354269119b3bb6d9f4c97954faa/kratos/python_scripts/point_output_process.py#L75

this can give variables that are defined in the Applications that are loaded at that point